qiskit/converters/__init__.py

```diff
@@ -17,12 +17,27 @@
 
 .. currentmodule:: qiskit.converters
 
-.. autofunction:: circuit_to_dag
-.. autofunction:: dag_to_circuit
+QuantumCircuit 
+==============
```

Comment:
Ideas welcomed for a better header name

Reply:
`QuantumCircuit -> Circuit Components` ? But I don't love it, I think the current proposal is fair.